src/rules/ResourceSourceChecker.js

```diff
@@ -82,8 +83,9 @@ class ResourceSourceChecker extends Rule {
             let match = re.exec(strippedSrc);
             while (match) {
                 let value = {
-                    severity: "error",
+                    severity: _this.severity,
```

Comment:
Oh good catch

Reply:
FYI I've split this off into a separate commit 123215e and PR https://github.com/iLib-js/i18nlint/pull/26